src/modules/formBuilder/components/formTree/FormTreeLabel.tsx

```diff
@@ -91,6 +91,24 @@ const FormTreeLabel: React.FC<FormTreeLabelProps> = ({
         height: '48px',
       }}
     >
+      <ConfirmationDialog
+        open={!!errors}
+        title='Cannot delete'
+        onConfirm={() => setErrors(undefined)}
+        onCancel={() => setErrors(undefined)}
+        loading={false}
+      >
+        Cannot delete "{labelProps.children}," because one or more other
+        questions depend on it.
```

Comment:
Did this come out of Figma or is it off the cuff?

I would add some text like: "Please remove all conditional dependencies from dependent items and try again. "

Reply:
Per Bobby's designs, adding similar: "All references to this item must be removed before it can be deleted."